docs/UserGuide.md

```diff
@@ -407,15 +408,34 @@ For the list of tab indexes to use:
 >
 > [IMAGE COMING SOON]
 
+<br>
 <br>
 
-#### 👀Viewing Help : `help`
+#### Clearing all Entries : `clear`
 
-You can view a message explaining how to access the help page, if you require assistance in understanding the commands and their formats.
+You can clear all students from F.A.K.E.J.A.R.V.I.S. if you need to reset the student database or remove all existing student records.
 
-![help message](images/helpMessage.png)
+Format: `clear`
 
-Format: `help`
+<br>
+
+<div class="alert alert-warning">
+<md>
+:warning: **Warning**:
+
+* This action is irreversible
+  </md> </div>
+
+<br>
+
+> **📖Example 1:**
+>
+> **Input:** `clear` Deletes all students in F.A.K.E.J.A.R.V.I.S.
+>
+> **Output:**
+>`Address book has been cleared!`
```

Comment:
Just occurred to me, should we change Address Book to F.A.K.E.J.A.R.V.I.S here?
i.e update UG and code. Seems like a bug sia.

Reply:
I'm not sure whether we can for this though. I think its fine?